Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:20
+FROM node:20.12
```

Comment:
CI is very specific about version numbers.  Would it make sense for the version here to match the version tested against?

Reply:
Our current approach for Docker images is to fix the minor version and let them get patches. That means in case of significant security issues in the image a rebuild is sufficient and we don’t have to rerelease. 

The downside is that the build isn’t entirely reproducible and, like you say, doesn’t exactly match CI which needs an explicit patch version. The alternative would be to use SHA hashes for images. 

I think where we’ve landed is a good balance. Maybe we go with this for now to match other parts of the system and we can have a broader discussion if you think we should go back to tighter reproducibility.